NitroxModel/Platforms/Store/Steam.cs

```diff
@@ -149,15 +149,64 @@ await RegistryEx.CompareWaitAsync<int>(@"SOFTWARE\Valve\Steam\ActiveProcess\Acti
             throw new GamePlatformException(this, "Timeout reached while waiting for platform to start. Try again once platform has finished loading.", ex);
         }
 
-        return ProcessEx.From(CreateSteamGameStartInfo(pathToGameExe, GetExeFile(), launchArguments, steamAppId, skipSteam));
+        if (bigPictureMode)
+        {
+            if (isPlatformStartingUp)
+            {
+                // TODO: Instead of waiting, detect when Steam Big Picture is ready to be started by Steam.
```

Comment:
Do we want to do the TODO now or are we happy to do this in the future?

Reply:
Gave a quick glance at the Steam logs but nothing big picture specific. I'll leave it for the future.